packer/scylla_install_image

```diff
@@ -118,14 +118,17 @@ if __name__ == '__main__':
 
         kernel_opt = ''
         grub_variable = 'GRUB_CMDLINE_LINUX_DEFAULT'
+        run('systemctl mask amazon-ssm-agent', shell=True, check=True)
     elif args.target_cloud == 'gce':
         # align with other clouds image
         run('apt-get purge -y rsyslog', shell=True, check=True)
         kernel_opt = ''
         grub_variable = 'GRUB_CMDLINE_LINUX_DEFAULT'
+        run('systemctl mask google-osconfig-agent', shell=True, check=True)
     elif args.target_cloud == 'azure':
         kernel_opt = ' rootdelay=300'
         grub_variable = 'GRUB_CMDLINE_LINUX'
+        run('systemctl mask walinuxagent', shell=True, check=True)
```

Comment:
@soyacz 

Aren't we using this for something in SCT ?

Reply:
yes, we use it for rebooting with running OS command (`reboot -f`) due issue with SDK (https://github.com/scylladb/scylla-cluster-tests/pull/6628)
We would need to adjust SCT to enable it back when VM is provisioned.

Reply:
@soyacz Can you open an issue or send a fix? we should probably also have it for 2025.1 since this is about to be backported

Reply:
Thanks @soyacz , I assume we can move forward with this PR then ?